index.html

```diff
@@ -31,21 +31,22 @@
         <div class="container-nav">
           <a href="index.html" class="logo-title">Marlyn Mayienga</a>
           <nav class="nav-bar">
-            <ul>
+            <ul class="menu">
               <li class="portfolio">
-                <a href="#portfolio">Portfolio </a>
+                <a class="menuItem" href="#portfolio">Portfolio </a>
               </li>
               <li>
-                <a href="#">About </a>
+                <a class="menuItem" href="#">About </a>
```

Comment:
- [ ] As the previous reviewer mentioned, the mobile menu options should take you to the proper section and the page, kindly fix this link so it links to the About section